src/zenml/zen_server/deploy/helm/README.md

```diff
@@ -0,0 +1,58 @@
+# ZenML Helm Chart
+
+![ZenML Logo](https://raw.githubusercontent.com/zenml-io/zenml/main/docs/book/.gitbook/assets/zenml_logo.png)
+
+## Overview
+
+ZenML is an open-source MLOps framework designed to help you create robust, maintainable, and production-ready machine learning pipelines.
+
+## Features
+
+- Easy deployment of ZenML server on Kubernetes.
+- Various authentication schemes including OAuth2 and HTTP Basic.
+- Highly configurable via Helm values.
+- Supports multiple secrets store backends like AWS Secrets Manager, GCP Secrets Manager, and Azure Key Vault.
+
+## Quickstart
+
+### Install the Chart
+
+To install the ZenML chart directly from Amazon ECR, use the following command:
+
+```bash
+# example command for version 0.44.2
+helm install my-zenml oci://public.ecr.aws/zenml/zenml --version 0.44.2
+```
+
+Note: Ensure you have OCI support enabled in your Helm client and that you are authenticated with Amazon ECR.
+
+## Configuration
+
+This chart offers a multitude of configuration options. For detailed information, check the default [`values.yaml`](values.yaml) file.
+
+### Basic Options
+
+- `zenml.replicaCount`: Number of ZenML server replicas.
+- `zenml.image.repository`: ZenML server image repository.
+- `zenml.authType`: Authentication type (e.g., `NO_AUTH`, `HTTP_BASIC`, `OAUTH2_PASSWORD_BEARER`).
```

Comment:
These two are not so important, but I would mention these instead:

* `zenml.defaultUsername`
* `zenml.defaultPassword`